tools/gfastats/gfastats.xml

```diff
@@ -44,6 +44,15 @@
                     --line-length $mode_condition.output_condition.line_length
                 #end if
             #end if
+            #if $mode_condition.output_condition.out_format == 'gfa'
+                #if $mode_condition.output_condition.terminal_overlaps_condition.terminal_overlaps_select == 'yes':
+                --discover-terminal-overlaps $mode_condition.output_condition.terminal_overlaps_condition.terminal_overlaps_length
+                #end if
+            #else if $mode_condition.output_condition.out_format == 'gfa.gz'
```

Comment:
It might be the fever, but both the if and the else is the same, is that correct? of the we can remove the if

Reply:
of wite someting like `#if $mode_condition.output_condition.out_format in ['gfa', 'gfa.gz']`

Reply:
it's not the fever, you're right! just pushed a commit using that syntax for the `--discover-terminal-overlaps` case and the `--line-length` case, thank you! also feel better soon :(